bindata/bootkube/manifests/secret-bound-sa-token-signing-key.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: next-bound-service-account-signing-key
+  namespace: openshift-kube-apiserver-operator
+  annotations: {{- if .UserProvidedBoundSASigningKey}}
+    "auth.openshift.io/do-not-rotate": "true" {{end}}
```

Comment:
I just wanted to put a marker that this resource must not be rotated since this was user provided. I agree that the RSA key-pair does not have validitity constraints, but it in case future code adds capability to keep rotating the keys for increases security posture this annotation could be used to make sure we don't adopt rotation of user provided keys..

Reply:
> I just wanted to put a marker that this resource must not be rotated since this was user provided. I agree that the RSA key-pair does not have validitity constraints, but it in case future code adds capability to keep rotating the keys for increases security posture this annotation could be used to make sure we don't adopt rotation of user provided keys..

I call YAGNI, but in the event that differentiating a user-supplied key vs a generated one is desirable I suggest indicating a more fundamental property (i.e. `auth.openshift.io/user-supplied-bound-token-key`) rather than predicting future requirements.

Reply:
Fixed this to use the annotation you mentioned. see 46e42cf